roles/repo_setup/tasks/configure.yml

```diff
@@ -5,6 +5,7 @@
   when:
     - content_provider_dlrn_md5_hash is defined
     - content_provider_dlrn_md5_hash | length > 0
+    - not cifmw_run_update|default(false)
```

Comment:
Hum... so we don't want to consume the content_provider hash during update.  So what about jobs that might use content provider, won't that be a problem ?

Reply:
For some jobs we may want to use content provider, but not during deployment phase.
For deployment we need to fetch stable content published and use content provider only during update playbook run. cifmw_run_update is defined in update job under vars, so I need to move it to new update extras vars